sql/4_2_2-to-5_0_0_upgrade.sql

```diff
@@ -2137,3 +2141,21 @@ CREATE TABLE `product_registration` (
 #IfNotRow2D list_options list_id Eye_Defaults_for_GENERAL option_id LADNEXA
 INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`) VALUES ('Eye_Defaults_for_GENERAL', 'LADNEXA', 'normal lacrimal gland and orbit', 91, 0, 0, '', 'EXT', '', 0, 0, 0, '');
 #EndIf
+
+-- Table to copy log contents for audit log tamper resistance check
+#IfNotTable log_validator
+CREATE TABLE `log_validator` (
+  `log_id` bigint(20) NOT NULL,
+  `log_checksum` longtext DEFAULT NULL,
+  PRIMARY KEY (`log_id`)
+) ENGINE=InnoDB;
+
+-- Add Image Result note type for electronic indication - a12
+#IfNotRow2D list_options list_id note_type option_id Image Results
+	SET @seq = (SELECT MAX(seq) FROM  list_options where list_id = 'note_type');
+	INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`,`is_default`) VALUES ('note_type', 'Image Results', 'Image Results', @seq+5, 0);
+#EndIf
+
+#IfMissingColumn pnotes document_id
+	ALTER TABLE `pnotes` ADD `document_id` int(11) DEFAULT NULL;
```

Comment:
It is done via the setGpRelation() function. Do a grep for it to see how it's used.